data/language/en-GB.txt

```diff
@@ -4117,6 +4117,8 @@ STR_5808    :{WINDOW_COLOUR_2}Number of shops and stalls: {BLACK}{COMMA16}
 STR_5809    :{WINDOW_COLOUR_2}Number of information kiosks and other facilities: {BLACK}{COMMA16}
 STR_5810    :Disable train length limit
 STR_5811    :{SMALLFONT}{BLACK}If checked, you can have up to{NEWLINE}255 cars per train
+STR_5812    :{SMALLFONT}{BLACK}Refurbishes the ride
```

Comment:
Or display it in a confirmation prompt, and if there is one then the button does not have to be disabled either.